bundles/http/Citrix/0b4d1a6b-30f3-4134-a354-42fa008b757e/f3b61af4-efd0-43c2-9a08-a467fc8242c1.js

```diff
@@ -0,0 +1,1294 @@
+let parser = library.load("xml2js")
+let jp = library.load("jsonpath-plus")
+const moment_lib = library.load("moment-timezone")
+
+async function logIn(client, userName, password) {
+    let session = [];
+    const response = await client.fetch('/wfc/XmlService', {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/xml",
+            "Accept": "application/xml"
+        },
+        body: `<KRONOS_WFC VERSION="1.0"><REQUEST OBJECT="SYSTEM" ACTION="LOGON" USERNAME="${userName}" PASSWORD="${password}"/></KRONOS_WFC>`
+    });
+
+    const cookies = await response.headers.get('set-cookie');
+    const data = await response.text();
+    parser.parseString(data, (err , result)=>{
```

Comment:
You can use [promise version](https://github.com/Leonidas-from-XIV/node-xml2js#promise-usage) too.
IMHO it would be much more readable when you use async/await everywhere.